new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/Domain.java

```diff
@@ -105,7 +107,7 @@ public static boolean shutdown(String domainUid, String namespace) {
    * @return true if patching domain resource succeeded, false otherwise
    */
   public static boolean restart(String domainUid, String namespace) {
-
+    LoggingFacade logger = getLogger();
```

Comment:
can we move the logger definition to the top so we don't need to make two logger definition here.

Reply:
do you mean make it class level?

Reply:
yes

Reply:
as explained in slack, can't use the class level variable here, it has to be local variable to get the logger from current thread